### ISSUE TITLE
Temperatures in Fahrenheit show Celsius values instead

### ISSUE THREAD

**Issue by keith24** [martykan/forecastie#646]
When set to fahrenheit, it always shows the units as `°F` but the number is obviously celcius.  Doesn't always happen but if you open and close the app a few times you should be able to reproduce.

**Comment by robinpaulson** [comment]
Interesting, I couldn't reproduce this bug. Could you tell me which location you're in, i wonder if that is the cause?

**Comment by keith24** [comment]
I woke up this morning and it's correct again.  I'm in Conejos, Colorado.  I have a thermometer and it was showing ~28°F yesterday and the app was showing `-0.5°F`.  Now both are showing 30°F.  I'm not sure when it changed, but the taskbar is affected too so I will try to watch that and try to determine what triggers it.  Is there a way to access logs without installing a bunch of android sdk packages on my computer?

**Comment by robinpaulson** [comment]
Is it possible this only happens when the temperature is in negative numbers on the centigrade scale?

**Comment by keith24** [comment]
Doesn't look like it.  

I just set the location to `Fairbanks, Alaska` and it shows `-2.5°F`.  Actual weather is `1°F/-17°` according to the [NWS](https://forecast.weather.gov/MapClick.php?lon=-147.761&lat=64.837).

**Comment by robinpaulson** [comment]
> Doesn't look like it.
> 
> I just set the location to `Fairbanks, Alaska` and it shows `-2.5°F`. Actual weather is `1°F/-17°C`

that sounds like it confirms my hypothesis

**Comment by keith24** [comment]
Well `1°F` is pretty close to `-2.5°F` so I think it's showing the correct scale and not doing the bug thing.

**Comment by keith24** [comment]
Oh, you mean that the bug only happens when the temperature is *above* freezing?  No, it was showing ~33°F all day today which was accurate.

**Comment by keith24** [comment]
I've tried toggling all the settings and restarting the app and can't reproduce it.   There's a chance that I had the location set wrong or the new location hadn't detected yet.  I'm just going to close this for now and come back if I see it happen again.

**Comment by keith24** [comment]
It happened twice today, that I saw celcius numbers in the notification but as soon as I tapped on it, it updated to the correct fahrenheit before my own eyes.  

Oh, neat I can drag-and-drop videos in here, proving my sanity.  

https://user-images.githubusercontent.com/10123447/147864438-60382e1f-25e9-474e-b503-c2b81b0c5fc1.mp4

So `9°F` is correct.  It was never -12°F so this isn't a case of a delayed update.  Also 9°F = -12°C.
